rrule/src/wasm/mod.rs

```diff
@@ -0,0 +1,33 @@
+#![allow(
+    clippy::cast_possible_truncation,
+    clippy::doc_markdown,
+    clippy::unwrap_used
+)]
```

Comment:
These should all be removed. We don't wont to allow these things. Now that the wasm code is in the `rrule` crate the clippy lints in the root of the crate will be applied to this module